src/py_sonar_scanner/scanner.py

```diff
@@ -29,11 +30,16 @@ class Scanner:
 
     def __init__(self, cfg: Configuration):
         self.cfg = cfg
+        self.log = logging.getLogger(__name__)
+        self.log.setLevel(logging.INFO)
+        handler = logging.StreamHandler()
+        handler.terminator = ""
```

Comment:
I'm not sure I understand why we choose this terminator as opposed to `\n`. Could you perhaps clarify?

Reply:
So this was done already in the previous version of the code: `print(msg, end='')`. We do not need to add the `\n` at the end of the message because it is already part of the original message.